source/_integrations/ialarm.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: Antifurto365 iAlarm
+description: Instructions on how to integrate iAlarms alarms into Home Assistant.
+logo: antifurto365-ialarm.png
+ha_category:
+  - Alarm
+ha_iot_class: Local Polling
+ha_release: 0.119
+ha_domain: ialarm
+ha_codeowners:
+  - '@RyuzakiKK'
+---
+
+The `ialarm` platform provides connectivity with the [Antifurto365](https://www.antifurtocasa365.it/) iAlarm alarm systems.
+
+This platform supports the following services: `alarm_arm_away`, `alarm_arm_home` and `alarm_disarm`.
+
+## Configuration
+
+To enable this, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+alarm_control_panel:
+  - platform: ialarm
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>